plugins/build-env/pre-build-buildpack

```diff
@@ -6,8 +6,8 @@ source "$PLUGIN_AVAILABLE_PATH/config/functions"
 APP="$1"; IMAGE=$(get_app_image_name $APP); BUILD_ENV=""
 verify_app_name "$APP"
 
-[[ -z $(config_get --global CURL_CONNECT_TIMEOUT) ]] && config_set --global CURL_CONNECT_TIMEOUT=5
-[[ -z $(config_get --global CURL_TIMEOUT) ]] && config_set --global CURL_TIMEOUT=30
+[[ -z $(config_get --global WGET_CONNECT_TIMEOUT) ]] && config_set --global WGET_CONNECT_TIMEOUT=5
+[[ -z $(config_get --global WGET_TIMEOUT) ]] && config_set --global WGET_TIMEOUT=30
```

Comment:
These are internal environment variables for the buildpacks, so they should be reverted.